### DISCUSSION TITLE
openradioss multidomain computation?

### DISCUSSION THREAD

**Discussion by wanglei742** [OpenRadioss/OpenRadioss#2209] [Q&A]
Hi,All, 
i am using the ALE method for FSI, but for the usual way it will take a long time, so i resort to multidomain method to speed up my computation. it works well for official license, when i switched to openradioss solver, it prompts the "RAD2RAD" keyword error. so my question here does openradioss does not support the multidomain yet? 
thanks

**Comment by MarianBulla** [comment]
Hi Wanglei742,

and Happy New Year 2024 !

Unfortunately, RAD2RAD is not supported in OpenRadioss because it requires additional executable files that are not released.

Thanks and best regards,
Marian

**Reply by wanglei742** [reply]
Hi, Marian happy new year, thanks for your reply. for this question i am clear now~

**Reply by MarianBulla** [reply]
Hi Wanglei742,
how important is this feature for you ? Is it curcial/critical for you or do you have a good workaround for it ?
My question is related to the point, we can put into our focus for future developments of OpenRadioss.

Thanks and best regards,
Marian

**Reply by wanglei742** [reply]
Hi，Marian, 
it's quite important for me, i found the multidomain really speed up a lot even it's case by case. i am the official altair user, but our company has limited license/token to use and we have to share this resource especially at busy time. i believe i‘m not the singe case, others company also face the similar situation, so if more time effective function opens up then more users will come. 
thanks and the best regards.

**Reply by MarkusSchulten** [reply]
/RAD2RAD/ would be a very nice feature for efficient SPH calculations in OpenRadioss :)

**Reply by MarianBulla** [reply]
Hi @MarkusSchulten, 
Yes I agree with you.
But currently, it is only available for the commercial Radioss version.
So I will discuss this possibility for OpenRadioss, internally.
First have to see, how difficult it will be to open it, too.

Will let you know.

Thanks and best regards,
Marian

**Reply by Taylor1987** [reply]
Hi, Marian:
does the openradioss the multidomain now? if not, do you plan that in the near future?

**Reply by jan-skoda** [reply]
Hello!
I would like to know, if there is any progress in RAD2RAD thing for OpenRadioss?
I came across that, and it seems to be helpful for me..

Thank you!
Jan

**Reply by Nemo755** [reply]
Hi @MarianBulla,

Has there been any feedback internally on that discussion? 

I came here after following the rabbit hole through Altair's documentation  for multi domain analysis and through the repo looking for a compile target for an r2r executable for coupling. no pressure of course but having the direct and native coupling capability available would be incredibly helpful for research and modeling, especially at the scale where that coupling becomes most useful. Those of us with limited compute power for running these solvers in the first place are also unlikely to be able to afford a license for commercial Radioss. just my two cents. Thanks for your time if you look into this again, it's appreciated.

-Nemo755

**Reply by MarianBulla** [reply]
Hi @Nemo755,
the r2r approach is only in the commercial version of Simcenter Radioss available.
Thanks and best regards,
Marian

**Reply by Nemo755** [reply]
Hi Marian,
Thank you for the update.
Since this is repository is open source,  would you have have any objection to a custom built rad2rad coupler? I can branch and send you a pr in about a week's time. 
Best regards,
Chris
